### DISCUSSION TITLE
Allow the app to open the camera at runtime as well.

### DISCUSSION THREAD

**Discussion by dhaminitinAltran** [yasirkula/UnityNativeGallery#170] [Ideas]
Hi,
Thanks for such a great plugin I had this thought of giving the app flexibility to
allow the android app to open the camera at runtime take a picture and then use it in the application.

**Comment by yasirkula** [comment]
You can use [NativeCamera](https://github.com/yasirkula/UnityNativeCamera) for it.

**Reply by dhaminitinAltran** [reply]
OK, Thanks will it be possible to get the disk size/file size of the loaded image as well?

**Reply by yasirkula** [reply]
Both NativeGallery and NativeCamera return the image file's path, so you can call `new FileInfo(filePath).Length`.
